YMChat/src/main/assets/index.html

```diff
@@ -55,6 +56,11 @@
         }catch(e){
         }
 
+        var scriptUrl = customBaseUrl+"/widget/mobile.js"
+        if(version == '2'){
+        scriptUrl = customBaseUrl+"/widget/v2/mobile.js"
```

Comment:
just put one level of indentation on line 61

Reply:
done